openapi/paths/processes-dru/pPackage.yaml

```diff
@@ -0,0 +1,25 @@
+get:
+   summary: retrieve the formal description used to deploy a process
+   description: |
+      Access the formal description that can be used to deploy a process on an OGC API - Processes Server Instance.
+   operationId: getPackage
+   tags:
+     - Package
+   responses:
+     200:
+       description: The formal used to deploy a process
+       content:
+         application/ogcapppkg+json:
+            schema:
+              $ref: "../../schemas/processes-dru/ogcapppkg.yaml"
+         application/cwl:
+            schema:
+              $ref: "https://raw.githubusercontent.com/common-workflow-language/cwl-v1.2/1.2.1_proposed/json-schema/cwl.yaml"
+         application/cwl+json:
+            schema:
+              $ref: "https://raw.githubusercontent.com/common-workflow-language/cwl-v1.2/1.2.1_proposed/json-schema/cwl.yaml"
+         application/cwl+yaml:
+            schema:
+              $ref: "https://raw.githubusercontent.com/common-workflow-language/cwl-v1.2/1.2.1_proposed/json-schema/cwl.yaml"
```

Comment:
The URI is not valid anymore (to fix in other parts of the spec as well). 
I already flagged the issue here: https://github.com/common-workflow-language/cwl-v1.2/issues/278#issuecomment-2079981922

Waiting for their feedback for an official location.